constants/ClassGraph9.js

```diff
@@ -0,0 +1,80 @@
+export function Class9Graph () {
```

Comment:
Should this be a function? Or a const variable object that we export?

Reply:
In this case, it does not really matter.